README.md

```diff
@@ -3,8 +3,20 @@
 
 ![Ui](docs/images/Ui.png)
 
-* Placeholder text
+* This CLI application is designed for **Engineering Managers of tech companies.**
+
+    Example usages:
+  * Schedule an interview with an interviewee
+  * Detect clashes in interview timings
+  * Write notes about the interviewee
 * For the detailed documentation of this project, see the **[InterviewHub Product Website](https://se-education.org/addressbook-level3)**.
```

Comment:
The hyperlink destination should be updated to 
`https://ay2324s1-cs2103t-t11-2.github.io/tp/`

Reply:
- Yep, my apologies!